### PR TITLE
Update issue/pr templates

### PR DESCRIPTION
**Pull request by reyang**
## Changes

* Be more prescriptive in the issue templates.
* Minor improvements on the layout / markdownlint.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -4,21 +4,41 @@ about: Create a report to help us improve
...
-* Platform and OS version:
+# Bug Report
 
-If you're reporting a problem with a specific version of a library in this repo, please check whether the problem has been fixed on master.
+List of [NuGet packages](https://www.nuget.org/profiles/OpenTelemetry) and
+version that you are using:
 
-**Steps to reproduce.**
-Describe exactly how to reproduce the error. Include a code sample if applicable.
+* `OpenTelemetry 0.4.0-beta.2`
```
Comment (cijothomas) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]: did you meant this as an example?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -4,21 +4,41 @@ about: Create a report to help us improve
...
+
+Runtime version (e.g. net461, net48, netcoreapp2.1, netcoreapp3.1, etc. You can
+find this information from the `*.csproj` file):
+
+* `netcoreapp3.1`
```
Comment (cijothomas) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: was it meant to be an example?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (reyang) [reply .github/ISSUE_TEMPLATE/bug_report.md:12]: Yes, do you think we should leave it as blank?

Comment (cijothomas) [reply .github/ISSUE_TEMPLATE/bug_report.md:12]: Can you say something like (eg: OpenTelemetry 0.4.0-beta2) to make it obvious that its an example

Comment (reyang) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Good idea.

**Pull request merged**